build.ps1

```diff
@@ -0,0 +1,4 @@
+$location = $PSScriptRoot
+$build = Join-Path $location "build"
+CMake -S $location -B $build
+CMake --build $build
```

Comment:
Missing new-line.

Reply:
Also does this mean we're standardizing on PS for cross-plat build scripting?

Reply:
@jhendrixMSFT, @bsiegel  as far as I know, PS core is the only one that is installed by default on all Azure Dev Ops hosts (Windows, Linux, Mac OS X). This specific `build.ps1` is not used by CI and it's for developers. The problem is that by default CMake will output everything in current directory :-)

Reply:
IMO it's better to just not add these kinds of build scripts, but that's just me.